CondFormats/SiPixelObjects/src/SiPixelFrameConverter.cc

```diff
@@ -53,7 +53,7 @@ int SiPixelFrameConverter::toCabling(
   for  (IT it = path.begin(); it != path.end(); ++it) {
     const PixelROC * roc = theMap->findItem(*it); 
     if (!roc) return 2;
-    if (! roc->rawId() == detector.rawId) return 3;
+    if ((! roc->rawId()) == detector.rawId) return 3;
```

Comment:
Vincenzo,
If you give me a day I can fix it.
I also do not know what the intention was but I think I can figure it out.
I could have it ready by Monday.
Or shall I wait that this pull request is done?
Danek

On 3 Mar 2016, at 14:31, Vincenzo Innocente <notifications@github.com> wrote:

> In CondFormats/SiPixelObjects/src/SiPixelFrameConverter.cc:
> 
> > @@ -53,7 +53,7 @@ int SiPixelFrameConverter::toCabling(
> >    for  (IT it = path.begin(); it != path.end(); ++it) {
> >      const PixelROC * roc = theMap->findItem(*it); 
> >      if (!roc) return 2;
> > -    if (! roc->rawId() == detector.rawId) return 3;
> > +    if ((! roc->rawId()) == detector.rawId) return 3;
> 
> agree with @davidlt 
> move ( after the ! and close it after the last )
> 
> �
> Reply to this email directly or view it on GitHub.
>